core/trino-main/src/main/java/io/trino/sql/analyzer/StatementAnalyzer.java

```diff
@@ -865,6 +865,11 @@ protected Scope visitAnalyze(Analyze node, Optional<Scope> scope)
         @Override
         protected Scope visitCreateTableAsSelect(CreateTableAsSelect node, Optional<Scope> scope)
         {
+            // TODO: Remove when engine is supporting table replacement
+            if (node.isReplace()) {
+                throw semanticException(TABLE_ALREADY_EXISTS, node, "Replace table is not supported");
```

Comment:
The error token should be `NOT_SUPPORTED`. We don't know at this moment if table exists.